testproject/Assets/Tests/Runtime/MultiprocessRuntime/BaseMultiprocessTests.cs

```diff
@@ -141,6 +140,8 @@ public virtual void Teardown()
             MultiProcessLog("Running teardown");
             if (!IgnoreMultiprocessTests)
             {
+                MultiProcessLog($"Stop all workers after each test so we can start fresh ones.");
+                MultiprocessOrchestration.ShutdownAllProcesses();
```

Comment:
Remove this from the TearDown completely.
This is already done in this method:
    [OneTimeTearDown]
    public virtual void TeardownSuite()